TodoApp/TodoApp.Api.Tests/Controllers/ItemListController.test.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http;
+using NUnit.Framework;
+using TodoApp.Api.Controllers;
+using TodoApp.Api.Models;
+using TodoApp.Api.Tests.Comparers;
+
+namespace TodoApp.Api.Tests.Controllers
+{
+    internal static class ControllerExtensions
+    {
+        private static async Task<HttpStatusCode> ResolveResult(IHttpActionResult result)
```

Comment:
I suggest inlining this method into both public extension methods, reorganize individual statements and then extract common behavior again - method seems to rather obscure the execution instead of easing it.